include/allegro5/allegro_opengl.h

```diff
@@ -22,6 +22,8 @@
    extern "C" {
 #endif
 
+#include <stdint.h>
```

Comment:
Could you make this include allegro5/base.h instead? Technically we support platforms without stdint.h, and base.h takes care of the difference.